website/pages/docs/guides/dependencies.mdx

```diff
@@ -26,7 +26,7 @@ For example, in this guide we will pin to a concrete commit hash:
 
 ```toml copy filename="Scarb.toml"
 [dependencies]
-quaireaux = { git = "https://github.com/keep-starknet-strange/quaireaux.git", rev = "041379c" }
+quaireaux_math = { git = "https://github.com/keep-starknet-strange/quaireaux.git", rev = "041379c" }
```

Comment:
```suggestion
quaireaux_math = { git = "https://github.com/keep-starknet-strange/quaireaux.git", rev = "4c93251" }
```

let's update commit hash so that this example will work